src/blocks/stories/view.js

```diff
@@ -329,25 +338,13 @@ document.addEventListener( 'DOMContentLoaded', () => {
 	const slideID = location.hash.slice( location.hash.lastIndexOf( '-' ) + 1 );
 	const slide = document.getElementById( slideID );
 
-	if ( ! slide ) {
-		return;
-	}
-
-	const section = slide.closest( '.stories.carousel' );
-
-	if ( section ) {
+	if ( slide && slide.closest( '.stories.carousel' ) ) {
 		setTimeout( () => {
-			section.scrollIntoView( {
-				block: 'start',
-				behavior: 'smooth',
-			} );
-
-			const slides = slide.parentElement.children;
-			const slideIndex = [ ...slides ].findIndex(
+			const slideIndex = [ ...slide.parentElement.children ].findIndex(
 				( { id } ) => id === slideID
 			);
 
 			setSlide( slideIndex );
-		}, 200 );
+		}, 500 );
```

Comment:
This is really slow, and I don't love it, but there's a lot that needs to initialize before the heights can be calculated. I'm open to reworking this with a more intelligent approach, but I was hoping to just move on to other tasks.